config-tests/test_remote_settings.py

```diff
@@ -0,0 +1,58 @@
+import os
+import pytest
+
+from kinto_http import Client
+from kinto_http.patch_type import JSONPatch
+
+
+@pytest.mark.settings
+@pytest.mark.skipif(
+    pytest.config.getoption("env") != "stage",
+    reason="Test only runs on staging"
+)
+def test_create_collection_in_main_bucket(conf, env):
+    # Create a record inside the 'main-workspace' bucket and then
+    # request that a specific user review it
+    client = Client(
+        server_url=conf.get(env, "writer_server"),
+        auth=(os.getenv("RS_USER_1_LOGIN"), os.getenv("RS_USER_1_PASSWD"))
+    )
+    data = {"qa": "testing"}
+    client.create_collection(id="qatest", if_not_exists=True, bucket="main-workspace")
+    cr_response = client.create_record(data=data, collection="qatest", bucket="main-workspace")
+    new_record_id = cr_response["data"]["id"]
```

Comment:
nitpick: I would put the record creation after the group change

Reply:
Done